### ISSUE TITLE
TechStars-Saba Karim (Medium Post)

### ISSUE THREAD

**Issue by jaahmuhl** [promazo/Content-Team#50]

**Comment by jaahmuhl** [comment]
Transcript: https://www.rev.com/transcript-editor/shared/rexITeXXGfBojXoSjR1L-SPnZ7D0MyrAc9OIzaWpCnXRriWht3X7fpav030nT8xKqJhsN2_0yGunr-Yc8cFyej2kQPY?loadFrom=SharedLink

**Comment by jaahmuhl** [comment]
Raw Footage: https://drive.google.com/drive/folders/1k8I-SFKSlJiXGzygxCTeViKouuaD5lA3

**Comment by James-Moore18** [comment]
Medium

Caption: In a society that can make us feel somewhat isolated and lonely, helping others out of the kindness of your heart is becoming increasingly less common. However, as Saba mentions, we do not achieve great things or come across great opportunities without the help of someone else at some point along the way. Listen to Saba Karim, the Director of Startup Pipelines for TechStars, talk about why he helps people, why he believes helping others is so important, and the effect that helping people has had on his life in terms of helping him get to where he is. 

On screen Questions: Why is helping others so important and how can new graduates begin to do so even when they're early in their careers?
Clip: 38:47 - 46:26 is where it should end

Interviewer: Brett Hummel
Videographer: Bryan Acevedo
#promazo #reallifemba #promazoexecutiveinterviews #goals #dream #entrepreneur #leadership #ceo

**Comment by jaahmuhl** [comment]
@bryanace53  Hey Bryan I need to make an edit. Instead of ending it at 45:42 can you end it at 46:26?

**Comment by bryanace53** [comment]
https://drive.google.com/drive/u/4/folders/1SkKps3lDyWi7nQHGhHTxWM-L5iFpgPDt

**Comment by jaahmuhl** [comment]
@James-Moore18 When the video is a single stream, ie we don't have the other camera angle, you have to create the questions that you want to appear on the screen, so i need you to look at the parts where I am speaking and synthesize that into a short question.

**Comment by bryanace53** [comment]
https://drive.google.com/drive/u/4/folders/1SkKps3lDyWi7nQHGhHTxWM-L5iFpgPDt


Here's another medium post for SABA does his title change? Also you talked in this video. I was going to cut your parts off since we can see you but not sure if you wanted that so I will hold until told otherwise.

**Comment by jaahmuhl** [comment]
@bryanace53 

Yes please cut me out. I think it is around 2:32-3:12 & 5:45-6:36

**Comment by bryanace53** [comment]
https://drive.google.com/drive/u/4/folders/1SkKps3lDyWi7nQHGhHTxWM-L5iFpgPDt

I added the cuts you wanted from above. I added a dip to white so it's not a hard cut let me know if that works!

**Comment by jaahmuhl** [comment]
@bryanace53 I want to change all the dips to white with questions:

![2022-08-23_20-49-27.png](https://images.zenhubusercontent.com/5cb3b35dd74bd07972cea86e/d039d781-7840-4dc3-ad59-fd34b1b709b3)

**Comment by jaahmuhl** [comment]
@bryanace53 
Second Question:
![2022-08-23_20-55-11.png](https://images.zenhubusercontent.com/5cb3b35dd74bd07972cea86e/023e10d9-c18d-404a-bf22-a23b3126a8f7)

**Comment by bryanace53** [comment]
Here is the updated Saba video 

https://drive.google.com/drive/u/4/folders/1SkKps3lDyWi7nQHGhHTxWM-L5iFpgPDt

**Comment by bryanace53** [comment]
Hi what's the status on this as well.

**Comment by jaahmuhl** [comment]
@bryanace53 One last minor edit:

![2022-10-18_19-38-16.png](https://images.zenhubusercontent.com/5cb3b35dd74bd07972cea86e/184cd2f7-a57a-4ce4-8dcd-64748ab503f8)

**Comment by bryanace53** [comment]
https://drive.google.com/drive/u/4/folders/1SkKps3lDyWi7nQHGhHTxWM-L5iFpgPDt
